### ISSUE TITLE
Can't configure after reinstallation

### ISSUE THREAD

**Issue by szafran81** [LaggAt/hacs-govee#63]
## Version of the custom_component

v0.2.1

## Configuration

```yaml
don't have one

```

## Describe the bug
![image](https://user-images.githubusercontent.com/2608332/159584496-a5e7b5bd-27bd-4327-bf24-d915fe6c4908.png)

Basically I've had to uninstall the integration from HACS, and then reinstallation after reboot went fine, but when trying to add the integration into HA the above comes up (it says that the integration doesn't have a way to configure it via GUI)

## Debug log

```
2022-03-22 23:01:12 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading configuration flow for integration govee: cannot import name 'Govee' from 'govee_api_laggat' (/usr/local/lib/python3.9/site-packages/govee_api_laggat/__init__.py)
```
HA: v2022.3.6
If more info is needed then please say what is needed and I'll try to provide that.

**Comment by theastadler** [comment]
![image](https://user-images.githubusercontent.com/72948786/159611712-f1ec6a6d-a993-4482-90dd-b8a199310310.png)

I'm having the same issue.

**Comment by szafran81** [comment]
Fixed in Home Assistant 2022.3.7
